images/haproxy/Dockerfile

```diff
@@ -13,6 +13,6 @@
 # limitations under the License.
 
 # standard haproxy image + minimal config so the container will not exit
-ARG BASE="haproxy:2.1.7-alpine"
+ARG BASE="haproxy:2.2.0-alpine"
```

Comment:
any breaking changes we should know about?

Reply:
In our case (KIND project) I did not find any break change.

The new version includes some performance optimization, dynamic error handling, enhanced observability, etc.  In some cases, we can use it for debugging HA traffic.

Reply:
SGTM :-)